packages/terra-image/src/Image.jsx

```diff
@@ -12,6 +13,14 @@ const ImageVariant = {
   THUMBNAIL: 'thumbnail',
 };
 
+const ImageFitTypes = {
```

Comment:
FitTypes?

Reply:
Fixed it here https://github.com/cerner/terra-core/pull/2494/commits/36742f3d9719b0fe00bbada1fcb416a338e7253e.